setup.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
+from setuptools import Command, find_packages, setup
+
+# -----------------------------------------------------------------------------
+
+
+def system(command):
+    class SystemCommand(Command):
+        user_options = []
+
+        def initialize_options(self):
+            pass
+
+        def finalize_options(self):
+            pass
+
+        def run(self):
+            subprocess.check_call(command, shell=True)
+
+    return SystemCommand
+
+
+# -----------------------------------------------------------------------------
+
+setup(
+    name="tqp",
+    version='0.1.0',
+    description="An opinionated library for pub/sub over SQS and SNS",
+    url='https://github.com/4Catalyzer/tqp',
+    author="Giacomo Tagliabue",
+    author_email='giacomo@gmail.com',
+    license='MIT',
+    classifiers=(
+        'Development Status :: 3 - Alpha',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
```

Comment:
i don't think this is python 2 compatible

Reply:
yeah right. Are we fine dropping py2 compat?

Reply:
fine by me